avm/res/data-factory/factory/linked-service/main.bicep

```
@@ -0,0 +1,57 @@
+metadata name = 'Data Factory Linked Service'
+metadata description = 'This module deploys a Data Factory Linked Service.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('Required. The name of the Linked Service.')
+param name string
+
+@description('Required. The type of Linked Service. See https://learn.microsoft.com/en-us/azure/templates/microsoft.datafactory/factories/linkedservices?pivots=deployment-language-bicep#linkedservice-objects for more information.')
+param typeName string
+
+@description('Optional. Used to add connection properties for your linked services.')
+param typeProperties object = {}
+
+@description('Optional. The name of the Integration Runtime to use.')
+param integrationRuntimeName string
+
+@description('Optional. Use this to add parameters for a linked service connection string.')
+param customizedParameter object = {}
+
+@description('Optional. The description of the Integration Runtime.')
+param linkedServiceDescription string
```

```suggestion
param description string
```
There's a spec somewhere for it, but it essentially comes down to not replicating a service's name for parameter (unless the rest api has it too). The rational: If you have a parameter 'description' for a resource type 'linkedService', it should ideally be intuitive that it's the linkedService's description.